resources/public/api/conf/2.0/application.raml

```diff
@@ -616,4 +616,53 @@ traits:
                         description: The specified arrival does not exist, has been archived or is not available to the EORI number.
                         value:
                           code: NOT_FOUND
+        /{messageId}:
```

Comment:
It's failed here:

```json
{
    "statusCode":500,
    "message":"Underlying error while parsing YAML syntax: 'while parsing a block mapping in 'reader', line 519, column 7:  get: expected <block end>, but found BlockMappingStart in 'reader', line 619, column 9: /{messageId}:' --  [line=619, col=1]"
}
```

You can check to see how changes work locally by:

* Running this project via `sbt run`
* Run the `PREVIEW_RAML` service manager profile (ignore the ASSETS_FRONTEND failure)
* Go to http://localhost:9680/api-documentation/docs/preview?url=http%3A%2F%2Flocalhost%3A9487%2Fapi%2Fconf%2F2.0%2Fapplication.raml

If it renders, it's good, if not, you'll get an error. There is probably an incorrect indent somewhere...